src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -57,6 +60,21 @@ class BankAccountStep extends React.Component {
             routingNumber: props.achData.routingNumber || '',
             accountNumber: props.achData.accountNumber || '',
         };
+
+        this.errorTranslationKeys = {
```

Comment:
NAB, this is probably obvious but I think it would be good to leave a comment and establish that the keys of this map should be associated with the value in state.